web/src/lib/components/album-page/albums-controls.svelte

```diff
@@ -118,8 +118,8 @@
   options={Object.values(sortOptionsMetadata)}
   selectedOption={selectedSortOption}
   on:select={({ detail }) => handleChangeSortBy(detail)}
-  render={({ text }) => ({
-    title: text,
+  render={({ key }) => ({
```

Comment:
I don't think we should ever dynamically translate values. Just translate them in directly in the options list.

Reply:
Ok, I will fix it...

Reply:
There is a problem with translating text within `web/src/lib/utils/album-utils.ts`...
```ts
export const sortOptionsMetadata: AlbumSortOptionMetadata[] = [
  {
    id: AlbumSortBy.Title,
    text: $t('sort_title'),
    defaultOrder: SortOrder.Asc,
    columnStyle: 'text-left w-8/12 sm:w-4/12 md:w-4/12 md:w-4/12 xl:w-[30%] 2xl:w-[40%]',
  },
```
` Cannot format a message without first setting the initial locale.`

Therefore I used on the fly translation before.
Could you please help to fix that?

Reply:
A possible solution would be to remove `text` attribute and create mapping table in svelte with `AlbumSortBy` as a key and translated text as value and then use texts from this within `render`.

Reply:
It should be fixed, please check if this solution is ok.